Detectors/TPC/workflow/src/TPCTimeSeriesSpec.cxx

```diff
@@ -1001,7 +1027,7 @@ class TPCTimeSeries : public Task
     return true;
   }
 
-  void fillDCA(const gsl::span<const TrackTPC> tracksTPC, const gsl::span<const o2::dataformats::TrackTPCITS> tracksITSTPC, const gsl::span<const o2::dataformats::PrimaryVertex> vertices, const int iTrk, const int iThread, const std::unordered_map<unsigned int, std::array<int, 2>>& indicesITSTPC, const gsl::span<const o2::its::TrackITS> tracksITS)
+  void fillDCA(const gsl::span<const TrackTPC> tracksTPC, const gsl::span<const o2::dataformats::TrackTPCITS> tracksITSTPC, const gsl::span<const o2::dataformats::PrimaryVertex> vertices, const int iTrk, const int iThread, const std::unordered_map<unsigned int, std::array<int, 2>>& indicesITSTPC, const gsl::span<const o2::its::TrackITS> tracksITS, const std::vector<int>& idxTPCTrackToTOFCluster, const gsl::span<const o2::tof::Cluster> tofClusters)
   {
     TrackTPC track = tracksTPC[iTrk];
```

Comment:
I see that you are still cloning the full TPC track. Why? For the propagation it is enough to clone TrackParCov.

Reply:
Thanks, I thought since we want the errors we would still need then the full track. But you are right, thanks.